training_pipeline/pipeline/components/HFPusher/model_card.py

```diff
@@ -0,0 +1,28 @@
+from huggingface_hub import ModelCard, ModelCardData
+
+
+def create_card(card_data, template_path, model_metadata, 
+                placeholders, **template_kwargs):
+    """Creates model card.
+
+    Args:
+        model_metadata (dict): Dict of card metadata.
+        see here for what you can pass to the metadata section:
+        https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md
```

Comment:
```suggestion
        Refer to the link to know what you can pass to the metadata section:
        https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md
```

More inclusive. Not everyone can see.